haskell/linux/OS.hs

```diff
@@ -87,6 +96,18 @@ conf = ServerConfig
             lookup' #stickMap i \(x', y') -> onAxis x x' <> onAxis y y'
         ClientUpdate (SliderMove i x) ->
             lookup' #sliderMap i . onAxis $ x * 2 - 1
+        ClientUpdate (InputText _ t) -> do
+            (#extra % _2) .= t
```

Comment:
We should track each text input element separately. In fact, really the frontend should track this state anyway - see #35.